util/Makefile.in

```diff
@@ -164,6 +164,12 @@ clixon_util_validate: clixon_util_validate.c $(BELIBDEPS) $(LIBDEPS)
 clixon_util_dispatcher: clixon_util_dispatcher.c $(BELIBDEPS) $(LIBDEPS)
 	$(CC) $(INCLUDES) $(CPPFLAGS) -D__PROGRAM__=\"$@\" $(CFLAGS) $(LDFLAGS) $^ -l clixon_backend -o $@ $(LIBS) $(BELIBS)
 
+clixon_netconf_ssh_callhome: clixon_netconf_ssh_callhome.c $(LIBDEPS)
```

Comment:
Already covered by the generic APPS rules. Note not reason they should link to clixon, they are pure ssh apps.

Reply:
Reverted.